### PR TITLE
my submission

### PR DESCRIPTION
**Pull request by tsikatawill**

### CODE REVIEW

**Comment by blackmann** [comment]
Good start but your solution doesn't totally address the requirements.

Assume today is Tuesday, then we should have

Today
Tomorrow
Thu, 2nd June
Fri, 3rd June
Sat, 4th June

But your solution only address about today (exactly this Friday, tomorrow, next week and the rest)

You have 40 minutes, try to address this if you can

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,122 @@
-import activities from './resources/schedule.json'
+import activities from "./resources/schedule.json";
 import dayjs from "dayjs";
+import { useEffect, useState } from "react";
 
 function App() {
+  const today = new Date();
+  var tomorrow = new Date(today + 1);
+  tomorrow.setDate(new Date().getDate() + 1);
```
Comment (blackmann) [inline src/App.js]: I don't understand what's going on here. Can you explain?

**Comment by blackmann** [comment]
This is how the results looks. It doesn't match the requirements. The list shouldn't be numbered

Screen Shot 2022-06-03 at 10 14 40 PM

File: src/App.js [hunk 2]
```diff
@@ -1,23 +1,122 @@
...
+  const lastDay = new Date(
+    new Date().setDate(new Date().getDate() - new Date().getDay() + 6)
```
Comment (blackmann) [inline src/App.js:10]: This is smart

File: src/App.js (hunk 1 above)
Comment (tsikatawill) [reply src/App.js]: I've fixed it. Ill commit the new changes

File: src/App.js (hunk 2 above)
Comment (tsikatawill) [reply src/App.js:10]: Thank you

**Comment by blackmann** [comment]
Screen Shot 2022-06-03 at 10 45 23 PM

The title doesn't appear for this section's activities

**Pull request closed without merging**